hs_core/hydroshare/utils.py

```diff
@@ -432,17 +596,64 @@ def resource_file_add_process(resource, files, user, extract_metadata=False, **k
     return resource_file_objects
 
 
-def add_file_to_resource(resource, f):
+def add_file_to_resource(resource, f, fed_res_file_name_or_path='', fed_copy=None):
     """
     Add a ResourceFile to a Resource.  Adds the 'format' metadata element to the resource.
     :param resource: Resource to which file should be added
     :param f: File-like object to add to a resource
+    :param fed_res_file_name_or_path: the logical file name of the resource content file for federated iRODS resource
```

Comment:
@pkdash Thanks for catching this. Added doc string with detailed info on this parameter including the info I detailed above to make this parameter clear.